### PR TITLE
IBFT has immediate finality

### PR DESCRIPTION
**Pull request by siladu**
# Pull request checklist

https://consensys.github.io/doc.goquorum/PR-232/concepts/consensus/comparing-poa/

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [x] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-locally).

## Describe the change

Docs incorrectly stated that IBFT does not have immediate finality

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

## After creating your PR and tests have finished

Make sure that:

- [x] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [x] You've [previewed your changes on GitHub Pages](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-on-github-pages)

### CODE REVIEW

**Review by macfarla — APPROVED** [review]

File: docs/concepts/consensus/comparing-poa.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ Properties to consider when comparing the PoA consensus protocols are:
 
 ### Immediate finality
 
-QBFT and Raft have immediate finality.
-When using QBFT and Raft there are no forks and all valid blocks get included in the main chain.
+QBFT, IBFT and Raft have immediate finality.
+When using QBFT, IBFT and Raft there are no forks and all valid blocks get included in the main chain.
```
Comment (alexandratran) [inline docs/concepts/consensus/comparing-poa.md:38]:
```suggestion
QBFT, IBFT, and Raft have immediate finality.
When using QBFT, IBFT, and Raft there are no forks and all valid blocks get included in the main chain.
```

File: docs/concepts/consensus/comparing-poa.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ Properties to consider when comparing the PoA consensus protocols are:
...
 
-IBFT and Clique do not have immediate finality.
-Implementations using IBFT and Clique must be aware of forks and chain reorganizations occurring.
+Clique does not have immediate finality.
```
Comment (alexandratran) [inline docs/concepts/consensus/comparing-poa.md:40]:
```suggestion
Clique doesn't have immediate finality.
```

**Review by alexandratran — APPROVED** [review]
Thanks for catching this mistake! LGTM after minor stylistic suggestions.

**Review by bgravenorst — APPROVED** [review]
LGTM

**Pull request merged**